<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manipulator</class>
 <widget class="QDialog" name="Manipulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="qmlWgt" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <zorder>splitter</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="lbl_PosX">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="posX">
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_PosY">
      <property name="text">
       <string> Y:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="posY">
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
